src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+import { TextField } from "@radix-ui/themes";
+import SearchIcon from "/src/assets/search.svg";
+import Image from "next/image";
+export default function SearchBar() {
+  return (
+    <TextField.Root
+      size="3"
+      radius="full"
+      variant="soft"
+      className="w-[80em] h-14"
+      placeholder="Search fonts"
+      color="gray"
+    >
+      <TextField.Slot>
+        <Image src={SearchIcon} width={20} height={20} alt="logo" />
```

Comment:
`SearchIcon` indicates a React component. Should rename it to `searchIconUrl` to indicate a link

```suggestion
import searchIconUrl from "/src/assets/search.svg";
import Image from "next/image";
export default function SearchBar() {
  return (
    <TextField.Root
      size="3"
      radius="full"
      variant="soft"
      className="w-[80em] h-14"
      placeholder="Search fonts"
      color="gray"
    >
      <TextField.Slot>
        <Image src={searchIconUrl} width={20} height={20} alt="logo" />
```